packages/react/dismissable-layer/src/DismissableLayer.tsx

```diff
@@ -73,16 +73,22 @@ const DismissableLayer = React.forwardRef<DismissableLayerElement, DismissableLa
     } = props;
     const context = React.useContext(DismissableLayerContext);
     const [node, setNode] = React.useState<DismissableLayerElement | null>(null);
+    const [ownerDocument, setOwnerDocument] = React.useState<Document>(document);
```

Comment:
Do we need this in state, given that it can be derived from `node` which is already in state?

Reply:
You're right, it's not necessary. I've changed it to get it from `node`.